src/components/ListboxContainer.tsx

```diff
@@ -11,40 +11,51 @@ interface ListboxContainerProps {
 }
 
 const ListboxContainer = ({ layout, app, listboxOptions }: ListboxContainerProps) => {
-
-  const fieldName = getFieldName(layout);
+  const [fieldName, setFieldName] = useState<string | undefined>(undefined);
   const [listboxInstance, setListboxInstance] = useState<stardust.FieldInstance>();
-  const elRef = useRef();
+  const elRef = useRef<HTMLElement>();
+
+  useEffect(() => {
+    setFieldName(getFieldName(layout));
+  }, [layout?.title || layout?.qListObject?.qDimensionInfo?.qFallbackTitle]);
```

Comment:
Seems like we can simply remove this useEffect and add this to the listen array of below useEffect:
`getFieldName(layout)` like: `, [getFieldName(layout)]`

But why do we need to do this?